popup.js

```diff
@@ -35,3 +35,22 @@ function updateDistance(distance) {
     </div>
   `);
 }
+//pause tracking and add current wensiteDomain to isPaused array
+/*document.getElementById('pause-current-site')?.addEventListener('click', () => {
+  stats.pauseCurrentSite().then(() => {
+    header_elem.text('Paused')
+    chrome.tabs.query({ active: true, currentWindow: true }, (tabs) => {
+      chrome.tabs.sendMessage(tabs[0].id, { action: 'pause_tracking' });
+    });
+  });
+});*/
+document.getElementById('pause-current-site')?.addEventListener('click', () => {
+  chrome.tabs.query({ active: true, currentWindow: true }, (tabs) => {
+    const activeURL = new URL(tabs[0].url)
+    console.log(activeURL)
+    const websiteDomain = activeURL.hostname
+    stats.pauseCurrentSite(websiteDomain).then(() => {
+      header_elem.text('Paused')
+    })
```

Comment:
Let's move the tab querying logic to the `pauseCurrentSite` as the function name indicates it knows the current site

Reply:
Also, can we not replace the entire header with 'paused'. Maybe just add `(paused)` besides the exising header.